runner.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+
+BIN=$1
+shift 1
+
+echo $@
+
+java \
+	-XX:+IdleTuningGcOnIdle -Xtune:virtualized \
+	-jar "$BIN" \
+	run "$@"
```

Comment:
can you move "run" back into the Dockerfile? (leave this generic, not specific to our RRM CLI)